concepts/55 Vue Components/02 Create a DevExtreme Application/45 Authentication/10 Integrate with a Back End.md

```diff
@@ -0,0 +1,22 @@
+Stub authentication functions for back-end requests are located in the `src\auth.js` file. Update these functions to make actual requests to your back end.
+
+Each function returns an object with the following fields:
+
+<table class="dx-table">
+    <tr>
+        <th>Field</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>isOk</td>
+        <td>A Boolean value that is <b>true</b> if the request was successful and <b>false</b> otherwise.</td>
+    </tr>
+    <tr>
+        <td>message</td>
+        <td>An error message (if an error occurred).</td>
+    </tr>
+    <tr>
+        <td>data</td>
+        <td>The request result (user information).</td>
```

Comment:
I would say that it is unclear what kind of user info this field contains. Is this a custom field, or information about user who initialized the request (such as credentials, etc)?

Reply:
Is "information about the authenticated user" more clear?

Reply:
Yes, this is fine